packages/devtools_app/lib/src/screens/performance/performance_screen.dart

```diff
@@ -131,8 +133,60 @@ class WebPerformanceScreenBody extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return const Center(
-      child: Text('TODO: add instructions for using Chrome DevTools'),
+    final isFlutterWebApp =
+        serviceManager.connectedApp?.isFlutterWebAppNow ?? false;
+    return Markdown(
+      data: isFlutterWebApp ? flutterWebInstructionsMd : dartWebInstructionsMd,
+      onTapLink: (_, url, __) {
+        if (url != null) {
+          unawaited(
+            launchUrl(
+              Uri.parse(url),
+            ),
+          );
+        }
+      },
     );
   }
 }
+
+const timelineLink =
+    'https://api.flutter.dev/flutter/dart-developer/Timeline-class.html';
+const timelineTaskLink =
+    'https://api.flutter.dev/flutter/dart-developer/TimelineTask-class.html';
+const debugBuildsLink =
+    'https://api.flutter.dev/flutter/widgets/debugProfileBuildsEnabled.html';
+const profileModeLink = 'https://docs.flutter.dev/testing/build-modes#profile';
+const performancePanelLink =
+    'https://developer.chrome.com/docs/devtools/performance';
+
+const flutterWebInstructionsMd = '''
+# How to use Chrome DevTools for performance profiling
+
+The timeline events emitted by the Flutter framework as it works to build
+frames, draw scenes, and track other activity such as HTTP request timings and
+garbage collection are exposed in the Chrome DevTools performance panel.
+
+You can also emit your own timeline events using the `dart:developer` 
+[Timeline]($timelineLink) and [TimelineTask]($timelineTaskLink) APIs for further
+performance analysis.
+
```

Comment:
Can we add a more detailed section here?
`## Optional flags to enhance tracing`

We should these flags and a small description:
- debugProfileBuildsEnabled
- debugProfileBuildsEnabledUserWidgets
- debugProfileLayoutsEnabled
- debugProfilePaintsEnabled
-   HttpClient.enableTimelineLogging (we may need to check if this can be enabled for web since this is from dart:io...if it can't be enabled for web, then we should remove the reference to HTTP request timings in the above description)

Reply:
Added those! HttpClient.enableTimelineLogging can't be used for web apps, so removed from list (and from the above description)